lib/premailer/rails/css_loaders/cache_loader.rb

```diff
@@ -19,7 +19,7 @@ def clear!
         end
 
         def development_env?
-          defined?(::Rails) && ::Rails.env.development?
+          defined?(::Rails) && ::Rails.respond_to?(:env) && ::Rails.env.development?
```

Comment:
Would you mind wrapping this change and the one below so that the lines don't exceed 80 chars? Similarly to how it is above. Then I'll merge 😉